bundles/org.eclipse.equinox.security/src/org/eclipse/equinox/internal/security/storage/SecurePreferencesContainer.java

```diff
@@ -24,47 +24,44 @@
  * but a few notes on implementation design.
  * 
  * For a URL we get one secure preference tree that contains data. The top node
- * of that tree has a "root" type, it has special properties - the location, status,
- * and knowledge as to how persist the tree. 
+ * of that tree has a "root" type, it has special properties - the location,
+ * status, and knowledge as to how persist the tree.
  * 
- * On the user side - let's say we have the following preferences open via factory:
+ * On the user side - let's say we have the following preferences open via
+ * factory:
  * 
- * UserPreferences1 (Options1, URL1)
- * UserPreferences2 (Options2, URL1)
- * UserPreferences3 (Options3, URL2)
- * UserPreferences4 (Options4, URL2)
+ * UserPreferences1 (Options1, URL1) UserPreferences2 (Options2, URL1)
+ * UserPreferences3 (Options3, URL2) UserPreferences4 (Options4, URL2)
  * 
- * When we'll have 2 actual "back end" secure preferences tree with data: 
+ * When we'll have 2 actual "back end" secure preferences tree with data:
  * 
- * [UserPreferences1] -> [Options1] + 
- *                                    \
- *                                     [secure preferences1]   <- 1 : 1 -> URL1
- *                                    /
- * [UserPreferences2] -> [Options2] + 
+ * [UserPreferences1] -> [Options1] + \ [secure preferences1] <- 1 : 1 -> URL1 /
+ * [UserPreferences2] -> [Options2] +
```

Comment:
There was some ascii art that got broken by the reformat. Would you please find a way to preserver it.

Reply:
I've added <pre>-tags to the ASCII art. Please let me know if this is sufficient or if you have some other way in mind.

Reply:
Thanks, that's good enough.